### PR TITLE
Swap flying mobs to bees

### PR DESCRIPTION
**Pull request by hunterbell11**
Changed all flying mobs to bee's and hand coded all of their attacks to not die from stinger.

attempted to fix orby wandering off into a distance (i hope maybe not)
made orby slightly better at swimming if you bow him from a distance
made orby take 50% more melee damage but 50% less ranged damage to demotivate cheese

fixed flying mob item yoinking
fixed flying mob immortality
fixed protected area flying mobs being unkillable

made splitjaws spawn less in layer1-2 but slightly more in layer 3

### CODE REVIEW

File: servers/mobs/sync/plugins/MythicMobs/Skills/mineinabyss/flyingattack,yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+Damage_3:
```
Comment (0ffz) [inline servers/mobs/sync/plugins/MythicMobs/Skills/mineinabyss/flyingattack,yml]: File named incorrectly, comma instrad of .

File: servers/mobs/sync/plugins/Geary/mineinabyss/mobs/spawns/inverted_forest/spawn.inverted.inbyo.yml [hunk 2]
```diff
@@ -17,6 +17,10 @@
   allow:
     - BIRCH_LEAVES
     - OAK_LEAVES
+    - OAK_SLAB
+    - COBBLESTONE_SLAB
     - DARK_OAK_LEAVES
     - SPRUCE_LEAVES
     - OAK_WOOD
+    - COBBLESTONE
```
Comment (0ffz) [inline servers/mobs/sync/plugins/Geary/mineinabyss/mobs/spawns/inverted_forest/spawn.inverted.inbyo.yml]: Why allow this? Supposed to only spawn in trees

File: servers/mobs/sync/plugins/MythicMobs/Mobs/mineinabyss/allaytest.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+dosetori:
+  Type: ALLAY
```
Comment (0ffz) [inline servers/mobs/sync/plugins/MythicMobs/Mobs/mineinabyss/allaytest.yml:2]: Supposed to be a bee here?

**Review by 0ffz — CHANGES_REQUESTED** [review]

File: servers/mobs/sync/plugins/MythicMobs/Mobs/mineinabyss/allaytest.yml (hunk 3 above)
Comment (hunterbell11) [reply servers/mobs/sync/plugins/MythicMobs/Mobs/mineinabyss/allaytest.yml:2]: nope its a test mob

File: servers/mobs/sync/plugins/Geary/mineinabyss/mobs/spawns/inverted_forest/spawn.inverted.inbyo.yml (hunk 2 above)
Comment (hunterbell11) [reply servers/mobs/sync/plugins/Geary/mineinabyss/mobs/spawns/inverted_forest/spawn.inverted.inbyo.yml]: due to layer 2 grief thgeirs more bridges than wood now, enabling slabs makes them spawn on bridges again and can have them gate keep bridges and is a pretty cool feature

**Pull request merged**